packages/lib-content/src/screens/Home/DefaultHome/components/Researchers.js

```diff
@@ -0,0 +1,126 @@
+import { Anchor, Box, Paragraph } from 'grommet'
+import { useTranslation } from '../../../../translations/i18n.js'
+import styled, { css } from 'styled-components'
+import { SpacedHeading } from '@zooniverse/react-components'
+
+import SubHeading from '../../../../components/HeadingForAboutNav/SubHeading.js'
+import MaxWidthContent from '../../../../components/MaxWidthContent/MaxWidthContent.js'
+
+const StyledLink = styled(Anchor)`
+  width: 300px;
+  border-radius: 5px;
+  font-size: 0.8rem;
+  padding: 8px 5px;
+  text-align: center;
+  color: black;
+  background-color: ${props => props.theme.global.colors['neutral-2']};
+  font-weight: normal;
+  box-shadow: 1px 1px 8px rgba(0, 0, 0, 0.25);
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const Relative = styled(Box)`
+  position: relative;
+`
+
+const OverlayFade = styled(Box)`
+  position: absolute;
+  right: -1px;
+  top: 0;
+  width: 30px;
+  height: 100%;
+  ${props =>
+    props.theme.dark
+      ? css`
+          background: linear-gradient(
+            90deg,
+            rgba(255, 255, 255, 0) 0%,
+            ${props.theme.global.colors['dark-3']} 100%
+          );
+        `
+      : css`
+          background: linear-gradient(
+            90deg,
+            rgba(255, 255, 255, 0) 0%,
+            #fff 100%
+          );
+        `}
+`
+
+const VideoWrapper = styled(Box)`
+  min-width: 478px;
+  border-radius: 8px;
+  aspect-ratio: 16 / 9;
+  overflow: hidden;
+
+  @media (width <= 769px) {
+    min-width: 370px;
+  }
+`
+
+const YOUTUBE_SOURCES = [
+  'https://www.youtube-nocookie.com/embed/CaTNIoJy4Dg?si=-rG8luvWD_FtFBc5', // PH TESS
+  'https://www.youtube-nocookie.com/embed/8lZiZoBcMjE?si=wPKWlRjQcKeTIOB5', // Wildcam Gorongosa
+  'https://www.youtube.com/embed/B6diEbpEblQ?si=tt3jtcINdleiBT8g' // Backyard Worlds
```

Comment:
[minor] Any reason this one has no no-cookie? Making it a yes-cookie by boolean logic, I guess.